src/servers/http_server.cc

```diff
@@ -1111,6 +1112,22 @@ class HTTPAPIServer : public HTTPServerImpl {
   re2::RE2 cudasharedmemory_regex_;
 };
 
+void HTTPAPIServer::HandleInferErrors(evhtp_request_t* req, TRITONSERVER_Error* err) {
+  const char* message = TRITONSERVER_ErrorMessage(err);
+  std::string error_string(message);
+  EVBufferAddErrorJson(req->buffer_out, err);
+  if (error_string.find("GPU error occured") == std::string::npos) {
```

Comment:
`!=`?

Reply:
Should check with the customer on the exact error message to check

Reply:
I confirmed with customer